tools/internal_ci/linux/pull_request/grpc_distribtests_python.cfg

```diff
@@ -27,5 +27,5 @@ action {
 
 env_vars {
   key: "TASK_RUNNER_EXTRA_FILTERS"
-  value: "presubmit"
+  value: "presubmit -e aarch64 musllinux_1_1"
```

Comment:
I think the naming on this is a little messy. The name `TASK_RUNNER_EXTRA_FILTERS` implies that this env var will be passed as the `-f` argument to `task_runner.py` ([which it currently is](https://github.com/grpc/grpc/blob/4ef0173be8a0a4760cd98f8d0a73e5cb334489d4/tools/internal_ci/linux/grpc_distribtests_python.sh#L55)) but we're now storing `-e` args in here as well. I think we should either:

1. Split out a new `TASK_RUNNER_EXTRA_EXCLUDES` variable in addition to the existing `TASK_RUNNER_EXTRA_FILTERS` variable or
2. Create a combined `TASK_RUNNER_EXTRA_ARGS` variable where no flag is assumed.

But I think option #2 would be a more cross-cutting change affecting all languages and many jobs.

Reply:
Agreed. I attempted adding a new variable, but these are passed as Kokoro variables and require allowlisting in g3 first.

I'll add a TODO here for now. Please let me know if you consider this high priority.